### ISSUE TITLE
Logitech G Pro Wheel no longer works with ffbtools

### ISSUE THREAD

**Issue by kodatarule** [berarma/ffbtools#38]
After the latest commit it seems that Logitech G Pro Wheel no longer works with ffbwrap and the ffb has to be throttled to 19 otherwise it has delays and ffb is incorrect.

**Comment by berarma** [comment]
I think it should work now. Can you please test it?

**Comment by kodatarule** [comment]
Yes I can confirm the commit fixes the problem. Closing the issue.

**Comment by berarma** [comment]
Thanks!
